src/vmm/src/cpu_config/x86_64/cpuid/intel/normalize.rs

```diff
@@ -398,4 +422,106 @@ mod tests {
         assert!((leaf_7_0.result.ebx & (1 << 6)) > 0);
         assert!((leaf_7_0.result.ebx & (1 << 13)) > 0);
     }
+
+    #[test]
+    fn test_update_extended_topology_v2_entry_no_leaf_0x1f() {
+        let mut cpuid =
+            crate::cpu_config::x86_64::cpuid::IntelCpuid(std::collections::BTreeMap::from([(
+                crate::cpu_config::x86_64::cpuid::CpuidKey {
+                    leaf: 0xB,
+                    subleaf: 0,
+                },
+                crate::cpu_config::x86_64::cpuid::CpuidEntry {
+                    flags: crate::cpu_config::x86_64::cpuid::KvmCpuidFlags::SIGNIFICANT_INDEX,
+                    ..Default::default()
+                },
+            )]));
+
+        cpuid.update_extended_topology_v2_entry();
+
+        assert!(cpuid
+            .get(&CpuidKey {
+                leaf: 0x1F,
+                subleaf: 0,
+            })
+            .is_none());
+    }
+
+    #[test]
+    fn test_update_extended_topology_v2_entry() {
+        let mut cpuid =
+            crate::cpu_config::x86_64::cpuid::IntelCpuid(std::collections::BTreeMap::from([
```

Comment:
nit: I also think some `use` statements will make this more readable

Reply:
yeah, I followed the other bad tests :P will fix!